release/3.1/index.html

```diff
@@ -40,7 +40,7 @@ <h3 id="_installation">Installation</h3>
 <div class="ulist">
 <ul>
 <li>
-<p><a href="/3.1/Installing_Server_on_Red_Hat/index-katello.html">Installing Katello on RHEL/CentOS</a></p>
```

Comment:
This is generated from master: https://github.com/theforeman/foreman-documentation/blob/master/web/content/release-3.1.adoc. Please submit a PR for that or this will be reverted on the next push.